fairmq/sdk/DDSTask.h

```diff
@@ -27,19 +27,22 @@ class DDSTask
   public:
     using Id = std::uint64_t;
 
-    explicit DDSTask(Id id)
+    explicit DDSTask(Id id, Id collectionId)
         : fId(id)
+        , fCollectionId(collectionId)
     {}
 
     Id GetId() const { return fId; }
+    Id GetCollectionId() const { return fCollectionId; }
```

Comment:
This should return a `fair::mq::sdk::DDSCollection::Id` instead of a `fair::mq::sdk::DDSTask::Id` here.

Reply:
fixed